src/dotnet/APIView/APIViewWeb/HostedServices/ReviewBackgroundHostedService.cs

```diff
@@ -33,6 +36,18 @@ public ReviewBackgroundHostedService(IReviewManager reviewManager, IConfiguratio
             {
                 _autoArchiveInactiveGracePeriodMonths = 4;
             }
+            var backgroundTaskDisabledLangs = configuration["ReviewUpdateDisabledLanguages"];
+            if(!string.IsNullOrEmpty(backgroundTaskDisabledLangs))
```

Comment:
```suggestion
            if (!string.IsNullOrEmpty(backgroundTaskDisabledLangs))
```

Reply:
I had missed this comment. I will pull in this change in my next PR. thanks @konrad-jamrozik